__tests__/notifications.ts

```diff
@@ -1,26 +1,118 @@
 import {
+  authorizeRequest,
   disposeGraphQLTesting,
   GraphQLTestClient,
   GraphQLTestingState,
   initializeGraphQLTesting,
   MockContext,
 } from './helpers';
-import { Banner } from '../src/entity';
+import { Banner, Notification, NotificationType, User } from '../src/entity';
 import { DataSource } from 'typeorm';
 import createOrGetConnection from '../src/db';
+import request from 'supertest';
+import { FastifyInstance } from 'fastify';
+import { usersFixture } from './fixture/user';
 
+let app: FastifyInstance;
 let con: DataSource;
 let state: GraphQLTestingState;
 let client: GraphQLTestClient;
+let loggedUser: string = null;
 
 beforeAll(async () => {
   con = await createOrGetConnection();
-  state = await initializeGraphQLTesting(() => new MockContext(con));
+  state = await initializeGraphQLTesting(
+    () => new MockContext(con, loggedUser),
+  );
   client = state.client;
+  app = state.app;
+});
+
+beforeEach(async () => {
+  loggedUser = null;
 });
 
 afterAll(() => disposeGraphQLTesting(state));
 
+describe('notifications route', () => {
+  it('should return not found when not authorized', async () => {
+    await authorizeRequest(request(app.server).get('/notifications')).expect(
```

Comment:
This will probably fail the gateway. Should we perhaps return a default value there? Or on the contrary support 401 on the gateway side